AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -899,6 +890,51 @@ - (void)transitionContext:(_ASTransitionContext *)context didComplete:(BOOL)didC
   _transitionContext = nil;
 }
 
+#pragma mark - Layout
+
+- (void)_completePendingLayoutTransition
+{
+  ASDN::MutexLocker l(_propertyLock);
+  if (_pendingLayoutTransition) {
+    [self _setCalculatedLayout:_pendingLayoutTransition.pendingLayout];
+    [self _completeTransition:_pendingLayoutTransition];
+    _pendingLayoutTransition = nil;
+  }
+  [self _completeLayoutCalculation];
```

Comment:
I find the distinction between _completeTransition:, and _completeLayoutCalculation to be unclear and think these names need to be revised.

Also, _setCalculatedLayout: seems unusual to occur separately / before _completeLayoutCalculation.

Why does _completeLayoutCalculation occur even if _pendingLayoutTransition is nil?